owl/SOMA-OBJ.owl

```diff
@@ -1391,8 +1391,60 @@ In robotics, it is not uncommon to encounter shapes that are flipped compared to
     ///////////////////////////////////////////////////////////////////////////////////////
      -->
 
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#CanCut -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#CanCut">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#Variability"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#affordsBearer"/>
+                <owl:allValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA-OBJ.owl#Cutter"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#affordsTrigger"/>
+                <owl:allValuesFrom rdf:resource="http://www.ease-crc.org/ont/SOMA-OBJ.owl#CutObject"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:comment>The disposition of an object (the tool) to cut other objects. Such as a knife has cutting ability to cut a cucumber into pieces.</rdfs:comment>
+    </owl:Class>
+
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#CutObject -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#CutObject">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/SOMA.owl#AlteredObject"/>
+        <rdfs:comment>An object being cut down into pieces.</rdfs:comment>
+    </owl:Class>
+
+    <!-- http://www.ease-crc.org/ont/SOMA-OBJ.owl#Cutable -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#Cutable">
```

Comment:
I defined [Cuttability](https://github.com/kaviyachandran/situational_awareness_example/blob/master/owl/situation_awareness_example.owl#L711) as a disposition property of the objects that take the role of CutObject